### PR TITLE
Convert repo to TypeScript

### PR DESCRIPTION
**Pull request by MichaelDeBoey**

### CODE REVIEW

**Comment by SaraVieira** [comment]
oh sorry, I fucked up and merged a PR, can you please fix the conflicts? :(

File: src/components/Downtimes/fn.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { differenceInDays, isAfter, subDays } from 'date-fns';
+
+import { CheckWithDownTimes, CheckWithLastMonthDownTimes } from '../../types';
+
+const getEmptyArrayWithLength = (length: number) =>
```
Comment (SaraVieira) [inline src/components/Downtimes/fn.ts:5]: Nice!

**Review by SaraVieira — APPROVED** [review]
Looks awesome!


Thank yOU!

**Comment by MichaelDeBoey** [comment]
Will rebase 🙂

**Comment by MichaelDeBoey** [comment]
@SaraVieira Rebased 🙂

**Pull request merged**